lib/log_displayer.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+let cli   = require('heroku-cli-util');
+let https = require('https');
+let liner = require('../lib/line_transform');
+
+const COLORS = [
+  s => cli.color.cyan(s),
+  s => cli.color.magenta(s),
+  s => cli.color.yellow(s),
+  s => cli.color.green(s),
+  s => cli.color.red(s),
+  s => cli.color.blue(s),
+  s => cli.color.bold.cyan(s),
+  s => cli.color.bold.magenta(s),
+  s => cli.color.bold.yellow(s),
+  s => cli.color.bold.green(s),
+  s => cli.color.bold.red(s),
+  s => cli.color.bold.blue(s),
+];
+let assignedColors = {};
+function getColorForIdentifier (i) {
+  if (assignedColors[i]) return assignedColors[i];
+  assignedColors[i] = COLORS[Object.keys(assignedColors).length % COLORS.length];
+  return assignedColors[i];
+}
+
+let lineRegex = /^(.*?\[([\w-]+)([\d\.]+)?\]:)(.*)?$/;
+function colorize (line) {
+  let parsed = line.match(lineRegex);
+  if (!parsed) return line;
+  let header     = parsed[1];
+  let identifier = parsed[2];
+  let body       = parsed[4];
+  return getColorForIdentifier(identifier)(header) + body;
+}
+
+function readLogs (logplexURL) {
+  return new Promise(function (fulfill, reject) {
+    https.get(logplexURL, function (res) {
```

Comment:
@dickeyxxx Will not work behind https proxy, will work on extracting my library out into heroku-cli-util.